src/server/services/registrar.js

```diff
@@ -203,8 +204,43 @@ async function getShortname(offeredTermCode, classSectionID) {
   }
 }
 
+/**
+ * Get crosslisted courses from the registrar (stored as an array of shortnames)
+ *
+ * @param {string} offeredTermCode  Term.
+ * @param {string} classSectionID   ClassID aka SRS.
+ * @returns {Array}   Returns a list of crosslisted courses.
+ */
+async function getCrosslists(offeredTermCode, classSectionID) {
+  const crosslistXml = await axios(
+    `https://webservices.registrar.ucla.edu/SRDB/SRDBWeb.asmx/getConSched?user=${process.env.REG_DB_USER}&pass=${process.env.REG_DB_PASS}&term=${offeredTermCode}&SRS=${classSectionID};`
+  );
+
+  const crosslists = [];
+  xml2js.parseString(crosslistXml.data, (err, result) => {
+    result.ArrayOfGetConSchedData.getConSchedData.forEach(async crosslist => {
+      const crosslistShortname = await registrar.getShortname(
+        crosslist.term[0],
+        crosslist.srs[0]
+      );
+      if (crosslistShortname) {
+        crosslists.push({
+          term: crosslist.term[0],
+          srs: crosslist.srs[0],
+          shortname: crosslistShortname,
+        });
+      } else {
+        registrarDebug('getShortname returned null');
```

Comment:
@kilometerskang For this debugging info to be useful we need to know the term/srs in which it returned null